### PR TITLE
Migrate to new pages using plugins

### PR DESCRIPTION
**Pull request by lucasecdb**
Change the "injection" to work in the new builder of `pages` and `react`. This should not be merged, until vtex/checkout-instore#714 lands in master.

### CODE REVIEW

File: react/index.js [hunk 1]
```diff
@@ -1,10 +1,26 @@
...
 
-class StoneFields extends React.Component {
-  render() {
-    return null
-  }
+const StoneFields = () => {
+  return (
+    <Fragment>
+      <FormInput
+        ffor="customsenha"
```
Comment (lbebber) [inline react/index.js:8]: where do these values in `ffor` come from?

File: pages/plugins.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "store.instore-register > registration": "pre-registration.stone"
```
Comment (lbebber) [inline pages/plugins.json]: 🤷‍♂️

**Review by lbebber — APPROVED** [review]

File: pages/plugins.json (hunk 2 above)
Comment (lucasecdb) [reply pages/plugins.json]: ops

File: react/index.js (hunk 1 above)
Comment (lucasecdb) [reply react/index.js:8]: I don't know, really. I just copy and pasted the values that were in the `pages.json` before.

**Review by guilhermebruzzi — APPROVED** [review]
LGTM

**Pull request closed without merging**